app/views/runs/_split_table.slim

```diff
@@ -1,36 +1,139 @@
-            td.align-left.align-middle = segment.name.presence || '-'
-            td.align-right.align-middle
-              - if segment.gold?(timing)
-                .gold-split title='Gold'
-              - if segment.skipped?(timing) || segment.reduced?(timing)
-                span.time -
-              - else
-                span = format_ms(segment.duration_ms(timing))
-            - if segment.skipped?(timing)
-              td.align-right.time.align-middle -
+          td.align-right.align-middle = segment.segment_number + 1
+          td.align-left.align-middle = segment.name.presence || '-'
+          td.align-right.align-middle
+            - if segment.gold?(timing)
+              .gold-split title='Gold'
+            - if segment.skipped?(timing) || segment.reduced?(timing)
+              span.time -
             - else
-              td.align-right.align-middle = format_ms(segment.end_ms(timing))
-            - if segment.histories.any?
-              td.align-right
-                a.segment-graph-toggler.text-light href='#' data-segment=segment.id data-collapsed='1'
-                  = icon('fas', 'chart-pie')
-              tr hidden=true
-                td colspan='999' style='margin: 0; padding: 0;'
-                  .card.border-0.m-0.p-0
-                    .segment-graph-holder id="segment-graph-holder-#{segment.id}" data-segment=segment.id
+              span = format_ms(segment.duration_ms(timing))
+          - if segment.skipped?(timing)
+            td.align-right.time.align-middle -
+          - else
+            td.align-right.align-middle = format_ms(segment.end_ms(timing))
+          - if segment.histories.any?
+            td.align-right
+              a.segment-graph-toggler.text-light href='#' data-segment=segment.id data-collapsed='1'
+                - if danger
+                  .text-danger = icon('fas', 'chart-pie')
+                - else
+                  .text-primary = icon('fas', 'chart-pie')
+            tr hidden=true
+              td colspan='999' style='margin: 0; padding: 0;'
+                .card.border-0.m-0.p-0.bg-primary
+                  .card-group
+                    - if danger
+                      .card.bg-danger: .card-body
+                        h6.card-title
+                          ' Standard deviation
+                          - if run.belongs_to?(current_user)
+                            small.tip-right(
+                              title="A high standard deviation like this indicates inconsistency. You should practice
+                              this segment!"
+                            ) = icon('fas', 'exclamation-triangle')
+                          - else
+                            small.tip-right( title="A high standard deviation like this indicates inconsistency!")
+                              = icon('fas', 'exclamation-triangle')
+                        h4.card-subtitle = format_ms_casual(history_times.standard_deviation.to_i, 1)
```

Comment:
For runs with a messed up history (not sure why this one in particular is jacked up), this just shows an empty space.  Some kind of placeholder would be cool to maintain consistency with the rest of the boxes.

![chrome_2018-09-03_18-06-17](https://user-images.githubusercontent.com/7030913/45004767-241a4980-afa4-11e8-909b-f93a91350c5a.png)